src/rtc/h5_prep.py

```diff
@@ -18,9 +18,7 @@
 
 from nisar.workflows.h5_prep import set_get_geo_info
 
-BASE_HDF5_DATASET = f'/science/SENTINEL1'
-FREQ_GRID_SUB_PATH = 'RTC/grids/frequencyA'
-FREQ_GRID_DS = f'{BASE_HDF5_DATASET}/{FREQ_GRID_SUB_PATH}'
+FREQ_GRID_DS = '/data/frequencyA'
```

Comment:
What do you think about adding a brief comment for what `FREQ_GRID_DS` stands for?

Reply:
@seongsujeong , for some reason, github is messing with these comments. Please, keep in mind that xmin and xmax are referenced to the edges of the pixel, not center. So, if you have 2 pixels and the pixel spacing is 30, the xmax - xmin will be 60